go.mod

```diff
@@ -40,7 +40,7 @@ require (
 	github.com/ultraware/whitespace v0.0.4
 	github.com/uudashr/gocognit v0.0.0-20190926065955-1655d0de0517
 	github.com/valyala/quicktemplate v1.2.0
-	golang.org/x/tools v0.0.0-20190930201159-7c411dea38b0
+	golang.org/x/tools v0.0.0-20191010075000-0337d82405ff
```

Comment:
it includes my fix https://github.com/golang/tools/pull/162 for those who uses `go get`: no more `ssa` panics